server/service/src/programs/patient/update_patient.rs

```diff
@@ -53,6 +57,13 @@ fn generate(existing: NameRow, update: UpdatePatient) -> NameRow {
         last_name,
         gender,
         date_of_birth,
+        address1,
+        phone,
+        date_of_death,
+        is_deceased: match is_deceased {
+            Some(is_deceased) => is_deceased,
+            None => false,
+        },
```

Comment:
I just checked other update methods, e.g. in stocktake, here the parameter is an Option<bool> and then it is used like `u.is_locked = input_is_locked.unwrap_or(false);` so I guess for consistency do the same here.